src/webview/pipeline/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path');
+
+module.exports = {
```

Comment:
Do you need the lower level webpack for building the webview? 
If so, can you explain why you can't do it at the root level?

Reply:
We have several webviews, which build mostly independant SPA, I'm not sure how to do that with singe webpack config

Reply:
We put all the views in specific folder and just generate webpack config. This way you can build all the views running webpack once. 
https://github.com/redhat-developer/vscode-openshift-tools/blob/master/src/view/webpack.config.js